userspace/engine/rule_loader.cpp

```diff
@@ -36,34 +36,80 @@ using namespace std;
 using namespace libsinsp::filter;
 
 rule_loader::context::context(const std::string& name)
-	: name(name)
 {
+	location loc = {name, YAML::Mark(), "rules content", ""};
```

Comment:
Out of curiosity, why we do this here?

Reply:
Just so every context has some location, So for the top level context, it's a simple "somewhere in this file" location.